libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -248,6 +248,27 @@ void SRV_Channels::set_digital_outputs(uint16_t dig_mask, uint16_t rev_mask) {
     digital_mask |= dig_mask;
     reversible_mask |= rev_mask;
 
+    // add in NeoPixel and ProfiLED functions to digital array to determine anything else
+    // that should be disabled
+    for (uint8_t i = 0; i < NUM_SERVO_CHANNELS; i++) {
+        SRV_Channel &c = channels[i];
+        switch (c.function.get()) {
+            case SRV_Channel::k_LED_neopixel1:
```

Comment:
NeoPixels from scripting use the scripting output functions. The names ones added here are only if using notify.

Reply:
I'm not sure we can do anything about that. I suppose we could expose a helper that scripting could use to find out if a channel is disabled, but its getting really complicated. I think scripting needs to play ball here and only use channels that are assigned to it - the magic only goes so far.

Reply:
If we force scripting to use the neopixel outputs it prevents using notify and scripting at once for LED control.

Reply:
at the moment I don't think this prevents scripting working - the dangerous condition is where channels are not disabled and then you implicitly make them disabled by using scripting. That's not handled here and was not handled before either, so I guess the question is what to do. Either add a new scripting output or somehow reject the request.